internal/config.go

```diff
@@ -28,18 +28,56 @@ import (
 //
 //nolint:govet // we don't care about alignment there
 type Config struct {
```

Comment:
Let's update documentation to make it clear the difference between it and ImportConfig/ConfigFile